drivers/SmartThings/matter-window-covering/src/init.lua

```diff
@@ -123,29 +131,42 @@ local function handle_shade_level(driver, device, cmd)
   device:send(req)
 end
 
--- current lift percentage, changed to 100ths percent
-local function current_pos_handler(driver, device, ib, response)
-  if ib.data.value == nil then
-    return
-  end
-  local windowShade = capabilities.windowShade.windowShade
-  local position = 100 - math.floor((ib.data.value / 100))
-  device:emit_event_for_endpoint(ib.endpoint_id, capabilities.windowShadeLevel.shadeLevel(position))
-  if position == 0 then
-    device:emit_event_for_endpoint(ib.endpoint_id, windowShade.closed())
-  elseif position == 100 then
-    device:emit_event_for_endpoint(ib.endpoint_id, windowShade.open())
-  elseif position > 0 and position < 100 then
-    device:emit_event_for_endpoint(ib.endpoint_id, windowShade.partially_open())
-  else
-    device:emit_event_for_endpoint(ib.endpoint_id, windowShade.unknown())
+-- move to shade tilt level between 0-100
+local function handle_shade_tilt_level(driver, device, cmd)
```

Comment:
What events are generated for the `windowShade` capability for a tilt-only device? It looks like they are only generated when the lift position changes.

Reply:
If I understand your question correctly, the `windowShade` capability is now updated when either the lift position or the tilt level changes. See the changes made to `current_pos_handler`.